css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+body {
+  background-color: rgb(54 150 218);
+  color: white;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+img {
+  margin-right: 25%;
+  margin-left: 25%;
+  max-width: 75px;
+  max-height: auto;
+  object-fit: fill;
+  padding-top: 22px;
+  padding-right: 24px;
+}
+
+h2 {
+  text-align: center;
+  justify-content: center;
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-main {
+  flex-basis: 100%;
+  text-align: center;
+}
+
+.paddingtop {
+  padding-top: 8px;
+}
+
+.column {
+  flex-basis: calc(100% / 3);
+  align-content: center;
+}
```

Comment:
Column classes should only include `flex-basis` or `width`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.